components/form/Form.tsx

```diff
@@ -111,7 +111,7 @@ export interface FormComponentProps {
 
 // https://github.com/DefinitelyTyped/DefinitelyTyped/issues/9951
 export interface ComponentDecorator<TOwnProps> {
-  (component: React.ComponentClass<FormComponentProps & TOwnProps>): any;
+  (component: React.ComponentClass<FormComponentProps & TOwnProps>): React.ComponentClass<TOwnProps>;
```

Comment:
This change will cause `@Form.create()` throw errors, relate [this](https://github.com/ant-design/ant-design/pull/5560).